turbo.json

```diff
@@ -1,5 +1,6 @@
 {
   "$schema": "https://turborepo.org/schema.json",
+  "globalDependencies": ["tsconfig.json", "scripts/entrypoints.ts"],
```

Comment:
cc @juliusmarminge this seems to do it - looks good to you?

Reply:
sure!

Reply:
im still getting the dev errors though 
<img width="820" alt="CleanShot 2023-05-25 at 14 10 55@2x" src="https://github.com/trpc/trpc/assets/51714798/2dace5a6-bf0a-48a5-939c-3badb9ef7454">

Reply:
yeah, I tried to fix that but got deralied and fixed this instead :D